doozerlib/image.py

```diff
@@ -281,3 +277,8 @@ def get_additional_push_names(self, additional_registries):
                 push_names.append('{}/{}/{}'.format(registry, namespace, repo_name))
 
         return push_names
+
+def exec_latest_build_info(tag, component_name):
+    rc, stdout, stderr = exectools.cmd_gather(["brew", "latest-build", tag, component_name])
+    assertion.success(rc, "Unable to search brew builds: %s" % stderr)
```

Comment:
Please change this to a `.format()` notation. `%` is deprecated. (I realize you probably just copied it from elsewhere in here and that's ok... but I try to remove the old form anytime I come across one).